core/utils/svg.js

```diff
@@ -181,4 +181,4 @@ Svg.TEXT = new Svg('text');
  */
 Svg.TSPAN = new Svg('tspan');
 
-exports = Svg;
+exports.Svg = Svg;
```

Comment:
I'm a bit sceptical about this one: shouldn't all of the `Svg.*` constants be top-level named exports of this module?

Of course, it might be best to address that in a separate PR, so I've added this to the relevant list in #5073.